extensions/scarb-doc/tests/data/hello_world_no_features/src/hello_world-tests.md

```diff
@@ -4,7 +4,9 @@ Tests module
 
 Fully qualified path: [hello_world](./hello_world.md)::[tests](./hello_world-tests.md)
 
-## Free functions
-
-- [it_works](./hello_world-tests-it_works.md)
 
+Free functions
+ ---
+| | |
+|:---|:---|
+| [it_works](./hello_world-tests-it_works.md) | Reallyworks.[...](./hello_world-tests-it_works.md) |
```

Comment:
just letting you know that I'm aware that a space is missing :) 
👀 https://github.com/software-mansion/scarb/issues/1762#issuecomment-2768361397

Reply:
I was wrong. The problem is in the changes here. I'll fix it.